settings.py

```diff
@@ -895,6 +895,12 @@
 
 ES_SETTINGS = None
 
+CCZ_FILE_HOSTING_SUPPORTING_FILES = {
```

Comment:
not sure how I feel about having this in settings. Is there not a way to get them out the blob db directly without having to list them here? If you set the blob parent to the `CCZHostingLink` then it should be easy enough to list them all right?

Reply:
these files are shared by all links in a domain.
The process as of now is,
setup a file in blob db from the backend at the server and then make it available for HQ to show for all links in the domain. I thought id was the safest way to map it correctly and since this does not need flexibility i.e they are not going to be changed frequently (as of day they would change just when LTS updates) putting this in settings seemed the right choice without adding another model/complexity.

In a follow up some time down the line, we can consider making this configurable from the HQ interface itself where one can upload a file, give it a name, mark it as a supporting file/document and thats it. but as of now i thought it was better to just keep it simple.

Reply:
These are now removed and moved to a model